master/internal/audit.go

```diff
@@ -47,7 +47,7 @@ func auditLogMiddleware() echo.MiddlewareFunc {
 			}
 
 			for path := range staticWebDirectoryPaths {
-				if strings.HasPrefix(c.Path(), path) {
+				if strings.HasPrefix(c.Request().URL.Path, path) {
```

Comment:
@salonig23 this was a failing test I think because of this echo change

"Return an empty string for ctx.path if there is no registered path https://github.com/labstack/echo/pull/2385"

Using the request path is always correct here I think. Other usages of Path look to be correct with the new change.